packages/wayland_proxy_virtwl.rb

```diff
@@ -0,0 +1,38 @@
+require 'package'
+
+class Wayland_proxy_virtwl < Package
+  description 'Proxy Wayland connections across the VM boundary'
+  homepage 'https://github.com/talex5/wayland-proxy-virtwl'
+  @_ver = '5b576d5cad76de6ed7513cc85748efd78ca366cf'
+  version "#{@_ver}-1"
+  compatibility 'all'
+  source_url 'https://github.com/talex5/wayland-proxy-virtwl.git'
+  git_hashtag @_ver
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/5b576d5cad76de6ed7513cc85748efd78ca366cf-1_armv7l/wayland_proxy_virtwl-5b576d5cad76de6ed7513cc85748efd78ca366cf-1-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/5b576d5cad76de6ed7513cc85748efd78ca366cf-1_armv7l/wayland_proxy_virtwl-5b576d5cad76de6ed7513cc85748efd78ca366cf-1-chromeos-armv7l.tpxz',
+    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/5b576d5cad76de6ed7513cc85748efd78ca366cf-1_x86_64/wayland_proxy_virtwl-5b576d5cad76de6ed7513cc85748efd78ca366cf-1-chromeos-x86_64.tpxz'
```

Comment:
Were these binaries generated after the new versions of ocaml and opam were installed?

Reply:
These are going to be replaced... the new ocaml & opam using the new ocaml are a huge PITA...